# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../elvis-tiny/dont_install_as_vi.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
don't install $bindir/{vi,ex,view,input} because you are a
replacement, not the official thing
Alejandro

--- ./Makefile.orig	2005-02-10 00:49:31.000000000 -0300
+++ ./Makefile	2005-02-10 00:50:18.178348278 -0300
@@ -349,10 +349,6 @@
 	cp $(PROGS) $(BIN)
 	(cd $(BIN); chmod 755 $(PROGS))
 	(cd $(BIN); chown bin $(PROGS))
-	-ln $(BIN)/elvis $(BIN)/vi
-	-ln $(BIN)/elvis $(BIN)/ex
-	-ln $(BIN)/elvis $(BIN)/view
-	-ln $(BIN)/elvis $(BIN)/input
 
 inst.dos: $(DUMMY)
 	copy $(PROGS) $(BIN)
